examples/hello/hdf5Writer/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
+
+add_executable(hello_hdf5Writer_nompi helloHDF5Writer_nompi.cpp)
+message("    HDF5:   ${HDF5_INCLUDE_DIRS}")
+include_directories(${HDF5_INCLUDE_DIRS})
+target_link_libraries(hello_hdf5Writer_nompi adios2_nompi)
+
+if(ADIOS_BUILD_TESTING)
+  add_test(
+    NAME Example::hello::hdf5Writer_nompi
+    COMMAND hello_hdf5Writer_nompi
+  )
+endif()
```

Comment:
Let's leave the no-mpi version commented out for now since it's not included in the adios_nompi library